src/templates/grantCallForm.html

```diff
@@ -0,0 +1,10 @@
+<html>
+    <body>
+        <form action = "http://localhost:5000/grantcallResult" method = "post">
```

Comment:
```suggestion
        <form action = "/grantcallResult" method = "post">
```
`localhost` would link to the client computer. The LLM functions must be executed on the host, however.